app/readme.md

```diff
@@ -60,6 +60,46 @@ docker-compose up --build
 
 This will build the backend, database, and proxy, and start them up.
 
+If successful you will see something like this which indicates the backend is working
```

Comment:
"If successful you will see this message being repeated every few seconds which indicates that the backend is working"

Reply:
Right of course, good call